utxo/utxo.go

```diff
@@ -816,12 +817,14 @@ func (uv *UtxoVM) sortUnconfirmedTx() (map[string]*pb.Transaction, TxGraph, map[
 			txGraph[refTxID] = append(txGraph[refTxID], txID)
 		}
 	}
-	if len(txMap) > 0 {
-		avgDelay := totalDelay / int64(len(txMap)) //平均unconfirm滞留时间
-		uv.xlog.Info("average unconfirm delay", "micro-senconds", avgDelay/1e6, "count", len(txMap))
-		uv.avgDelay = avgDelay / 1e6
+	txMapSize := int64(len(txMap))
+	if txMapSize > 0 {
+		avgDelay := totalDelay / txMapSize //平均unconfirm滞留时间
+		microSec := avgDelay / 1e6
```

Comment:
If an expression is not used multiple times, no need to correct it.

Reply:
@af913337456 , any response to this comment?

Reply:
@yucaowang What's your suggestion about this? I'd like to know about it.

Reply:
@yucaowang In my opinion, my modifications include making the code look better.